Library/Formula/shml.rb

```diff
@@ -0,0 +1,16 @@
+class Shml < Formula
+  desc "Style Framework for The Terminal"
+  homepage "http://shml.xyz"
+  url "https://github.com/MaxCDN/shml/archive/1.0.4.tar.gz"
+  sha256 "583fefbbf2adf45d989b052696be0f20f695903878558a4d2e9f74255b91ed51"
+
+  def install
+    bin.install "shml.sh"
```

Comment:
I understand the point on two almost identical bins. But this is really something that is sourced as a framework, more than executed. It needs to be in the path to properly source within bash/zsh/etc. Recommended use is `source shml`, but as more common practice is to source `.sh` files. As such we really prefer to support both.